.travis.yml

```diff
@@ -1,5 +1,6 @@
 # Config file for automatic testing at travis-ci.org
 
+sudo: required
```

Comment:
why sudo?

Reply:
The tests were always terminated on travis. I guess we used to much memory on the containers. This solves this for now.